modules/ListView.js

```diff
@@ -319,15 +324,23 @@
     },
 
     /**
+     * Creates a DOM fragment with each child view appended in the order defined by
+     * modelsToRender(). This will clear the List View's DOM and invoke the necessary
+     * detach, register and render logic on each child view.
      * @return a DOM fragment with child view elements appended
-     * @method __buildChildViewsFragment
+     * @method __emptyAndRebuildChildViewsFragment
      * @private
      */
-    __buildChildViewsFragment: function(renderAlso) {
+    __emptyAndRebuildChildViewsFragment: function(renderAlso) {
       var injectionFragment = document.createDocumentFragment();
+      // Clearing the DOM will reduce the repaints needed as we detach each child view.
+      this.$el.empty();
```

Comment:
this feels like it should live outside of this method.  It is mutating the parent, where this method is about generating the child content.